active/RadioButtonsAPI.md

```diff
@@ -0,0 +1,105 @@
+# Background
+
+RadioButtons is our answer to enabling better keyboarding and narrator support for RadioButtons that need to be presented in a related list.
+
+Currently, the only way to group RadioButtons is through using a StackPanel and individually labeling each RadioButton through its GroupName property in order for narrator to read out the options somewhat understandably.
+
+However, using that method still left gaps in the narrator experience and StackPanel does not behave the way a user would expect when navigating the list via keyboard - RadioButtons was introduced to help solve those issues.
+
+# Description
+
+RadioButtons is a class extension of ListView that enables you to create related lists of RadioButtons easily, while also fully supporting keyboarding and narrator functionality.
+
+# Examples
```

Comment:
A big enhancement is this control's ability to work in an MVVM setup, could you show an example of databinding with this?

Reply:
You're referring to this ``` ItemsSource="{x:Bind radioButtonItems}" ``` kind of binding, correct?

Reply:
Right, and binding of the SelectedItem as well.

Reply:
Added.